corehq/apps/cloudcare/templates/cloudcare/new_app_version_modal.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+
+<div class="modal fade" id ="newAppVersionModal" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title">{% trans "A new version of this app is available" %}</h4>
+        </div>
+        <div class="modal-body">
+          <p>{% trans "Please click on \"Get Latest App\" to get on the newer version. It's recomended that you work on the latest version of the app." %}</p>
+          <p id="incompleteFormWarning" style="display: none;"> {% trans "Please note that all unsaved changes would be lost while moving to new version. You can submit the changes and manually refresh this page which will automatically take you to latest version."%}</p>
```

Comment:
nit: `blocktrans` would be best for these longer blocks of test

Reply:
Done https://github.com/dimagi/commcare-hq/pull/30778/commits/faa4c510044294fd2077aedc093f42445e81cc5e